sickchill.json

```
@@ -0,0 +1,54 @@
+{
+	"SickChill": {
+		"containers": {
+			"SickChill": {
+				"image": "linuxserver/sickchill",
+				"launch_order": 1,
+				"ports": {
+					"8081": {
+						"description": "Port for SickChill Web interface. Suggested default: 8081.",
+						"host_default": 8081,
+						"label": "WebUI port",
+						"protocol": "tcp",
+						"ui": true
+					}
+				},
+				"volumes": {
+					"/download": {
+						"description": "Choose a share to download to (also known as the sickchill monitor folder). It is important that the owner/group specified later have full access to the share (R/W/E).",
+						"label": "Download"
+					},
+					"/config": {
+						"description": "Choose a share to host the configuration settings for SickChill. It is important that the owner/group specified later have full access to the share (R/W/E).",
+						"label": "Config"
+					},
+					"/tv": {
+						"description": "Choose a share to host shows (also known as parent folder). You can add additional shares later via the RockOn's settings menu. It is important that the owner/group specified later have full access to the share (R/W/E).",
```

Minor style editing as I'm trying to harmonize the use of "rock-on":
```suggestion
						"description": "Choose a share to host shows (also known as parent folder). You can add additional shares later via the rock-on's settings menu. It is important that the owner/group specified later have full access to the share (R/W/E).",
```